### ISSUE TITLE
Model overfitting

### ISSUE THREAD

**Issue by haskarb** [aditya-AI/Information-Retrieval-System#1]
Hi Aditya!

Thanks for making your code public. I see that your training accuracy is higher, while your validation accuracy is much lower. Isn't your model overfitting?
